src/pages/workspaces/workspace/analysis/AnalysisLauncher.js

```diff
@@ -561,7 +561,14 @@ const PreviewHeader = ({
       // Status specific messaging which is not specific to an app
       Utils.cond(
         [_.includes(runtimeStatus, usableStatuses), () => h(StatusMessage, { hideSpinner: true }, ['Cloud environment is ready.'])],
-        [runtimeStatus === 'Creating', () => h(StatusMessage, ['Creating cloud environment. You can navigate away and return in 3-5 minutes.'])],
+        [
+          runtimeStatus === 'Creating' && isAzureWorkspace,
+          () => h(StatusMessage, ['Creating cloud environment. You can navigate away, this may take up to 10 minutes.']),
+        ],
+        [
+          runtimeStatus === 'Creating' && !isAzureWorkspace,
```

Comment:
should probably use `isGoogleWorkspace` here, not just the negation of the above condition